controller/pwm_profile/composite_test.go

```diff
@@ -0,0 +1,21 @@
+package pwm_profile
+
+import (
+	"testing"
+	"time"
+)
+
+func TestComposite(t *testing.T) {
+	conf := `
+{
+  "profiles":[
+	  {"type": "diurnal", "span": 300}, {"type": "fixed", "span": 150, "config":{"value": 0}}
+	]
+}
+`
+	p, err := Composite([]byte(conf), time.Now())
+	if err != nil {
+		t.Error(err)
+	}
+	p.Get(time.Now())
```

Comment:
Are there any assertions to make for this test?  It seems to just be testing that composite can unmarshal?

Reply:
yeah.. was done in a hurry. Adding assertions now